marketplacetests/payment/app.py

```diff
@@ -82,6 +83,9 @@ def tap_cancel_pin(self):
     def tap_pin_continue(self):
         button = self.marionette.find_element(*self._pin_continue_button_locator)
         Wait(marionette=self.marionette).until(lambda m: button.is_enabled())
+        # TODO: Even with the above wait, marionette often fails to tap on
+        # the button successfully. Adding a sleep fixes this for now.
+        time.sleep(1)
```

Comment:
If it's possible that the button is partially outside of the viewport then you could try the `scrollIntoView(false)` workaround here.